package.json

```
@@ -9,6 +9,7 @@
     "format": "prettier-standard \"**/*.{js,md,yml,json}\"",
     "test": "prettier-standard --lint --check \"**/*.{js,md,yml,json}\"",
     "test:integration": "ava \"test/integration/**/*.js\" -v --timeout=1m",
+    "test:regression": "ava \"test/regression/**/*.js\" -v --timeout=60m",
     "test:update-snapshots": "npm run test:integration -- --update-snapshots",
```

This script should be updated to this maybe?

```suggestion
    "test:update-snapshots": "npm run test:regression -- --update-snapshots",
```

---

Yes, that's a reasonable update! I'll have it sorted out in a sec =)